src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const oneDayRentCost = 40;
```

Comment:
Literals should be written in capital letters. (Lesson 'Transportation on Vacation Solution' , 2:17)
```suggestion
  const COST_PER_DAY = 40;
```